extensions/common/context.cc

```diff
@@ -198,51 +198,50 @@ TrafficDirection getTrafficDirection() {
   return TrafficDirection::Unspecified;
 }
 
-bool extractNodeFlatBuffer(const google::protobuf::Struct& metadata,
-                           flatbuffers::FlatBufferBuilder& fbb) {
+void extractEmptyNodeFlatBuffer(std::string* out) {
+  flatbuffers::FlatBufferBuilder fbb;
+  FlatNodeBuilder node(fbb);
+  auto data = node.Finish();
+  fbb.Finish(data);
+  out->assign(reinterpret_cast<const char*>(fbb.GetBufferPointer()),
+              fbb.GetSize());
+}
+
+bool extractPartialLocalNodeFlatBuffer(std::string* out) {
```

Comment:
This is partial only because the extensions uses the method does not need the full node info? The reason in the comment is `avoid parsing a protobuf from the host`. Not clear why that makes this need to be partially populated

Reply:
This is because we can't parse the list of labels without using protobuf. I linked to the issue. We'd need a different encoding for lists and maps but that's another ABI change. The partial node info is sufficient for the stats extension but not stackdriver extension, for example.

Reply:
Ok I see.